index.spec.js

```diff
@@ -0,0 +1,68 @@
+//import server
+const server = require('./api/server.js');
+
+//import supertest
+const request = require('supertest');
+
+
+describe('GET /games testing',()=>{
+
+  it('should respond with status code 200', async ()=>{
+    const response = await request(server).get('/games');
+    expect(response.status).toBe(200);
+  });
+
+  it('should return type JSON', async ()=>{
+    const response = await request(server).get('/games');
+    expect(response.type).toBe('application/json');
+  });
+
+  it('should return all games, or an empy array if no games', async ()=>{
```

Comment:
Good job taking the spec's requirements and communicating that via your test descriptions 👍